AIPscan/Reporter/helpers.py

```diff
@@ -145,3 +146,21 @@ def get_premis_xml_lines(file_object):
         premis_xml_lines = file_object.premis_object.split("\n")
 
     return premis_xml_lines
+
+
+def calculate_paging_window(page, pagination):
+    first_item = ((page - 1) * pagination.per_page) + 1
+    last_item = page * pagination.per_page
+
+    if last_item > pagination.total:
+        last_item = pagination.total
+
+    return first_item, last_item
+
+
+def urlencode_without_none_values(values):
+    for key in values:
+        if values[key] is None:
+            values[key] = ""
+
+    return urlencode(values)
```

Comment:
Maybe a good use case for `values.items()`, e.g.:

```python
def urlencode_without_none_values(values):
    """URL encode the values dict converting any None values to empty strings."""
    for key, value in values.items():
        if value is None:
            values[key] = ""
    return urlencode(values)
```

Here's an alternative using a dictionary comprenhension, I think this can be useful if you want to avoid mutating the input dictionary or doing it during iteration.

```python
def urlencode_without_none_values(values):
    """URL encode the values dict converting any None values to empty strings."""
    return urlencode(
        {
            key: (value if value is not None else '')
            for key, value in values.items()
        }
    )
```